server/rotation.go

```diff
@@ -125,8 +124,11 @@ func (k keyRotater) rotate() error {
 	var nextRotation time.Time
 	err = k.Storage.UpdateKeys(func(keys storage.Keys) (storage.Keys, error) {
 		tNow := k.now()
+
+		// if you are running multiple instances of dex, another instance
+		// could have already rotated the keys.
 		if tNow.Before(keys.NextRotation) {
-			return storage.Keys{}, errors.New("keys already rotated")
```

Comment:
can we have an info level log message here? Maybe explain that another server already rotated it?